tests/integration/targets/test_azure_load_balancer_with_public_ip/tasks/main.yml

```diff
@@ -12,6 +12,19 @@
     vars:
       test_resource_group: "{{ resource_group }}"
 
+  always:
+    - name: Cleanup - make sure Load Balancer deleted, resource_group={{ resource_group }}
+      azure.azcollection.azure_rm_loadbalancer:
+        resource_group: "{{ resource_group }}"
+        name: "{{ load_balancer_name }}"
+        state: absent
+
+    - name: Cleanup - make sure Public IP deleted, resource_group={{ resource_group }}
+      azure.azcollection.azure_rm_publicipaddress:
```

Comment:
I think that deleting the resource group here is a mistake
This is the pre-created resource group, am I right?